### PR TITLE
(fix): Modifying config to show-hide media metadata.

### PR DESCRIPTION
**Pull request by mdhaman**
- Changing config to show-hide media metadata
- Fixing the issue when a media item from monitoring is associated to an text item then metadata is not copied across to the associated item.

This would require change to the `superdesk.config.js` in the repo https://github.com/superdesk/superdesk.

### CODE REVIEW

File: superdesk.config.js [hunk 1]
```diff
@@ -8,7 +8,29 @@
 module.exports = function(grunt) {
     return {
         defaultRoute: '/workspace',
-        requiredMediaMetadata: ['headline', 'description_text', 'alt_text'],
+        requiredMediaMetadata: {
+            headline: {
```
Comment (petrjasek) [inline superdesk.config.js]: would probably do it like:
```
headline: true,
alt_text: true,
byline: false
```
given that the name is `requiredMediaMetadata`

Comment (mdhaman) [inline superdesk.config.js]: will change the name to something appropriate. what if we have `maxlength`?

Comment (petrjasek) [inline superdesk.config.js]: then it would make sense

**Comment by mdhaman** [comment]
@ioanpocol can please review this changes.

**Comment by ioanpocol** [comment]
Sorry for late review

**Pull request merged**